src/js/logger-ui-inspector.js

```diff
@@ -47,7 +43,7 @@ if (
 
 /******************************************************************************/
 
-const logger = globals.logger;
+const logger = self.logger;
```

Comment:
There might be a better way to share `logger` between the two files, but let's use `self` for now.